config/src/main/java/org/springframework/security/config/web/server/ServerHttpSecurity.java

```diff
@@ -2395,13 +2399,33 @@ public HeaderSpec contentSecurityPolicy(Customizer<ContentSecurityPolicySpec> co
 
 		/**
 		 * Configures {@code Feature-Policy} response header.
-		 * @param policyDirectives the policy directive(s)
+		 * @param policyDirectives the policy
 		 * @return the {@link FeaturePolicySpec} to configure
 		 */
 		public FeaturePolicySpec featurePolicy(String policyDirectives) {
 			return new FeaturePolicySpec(policyDirectives);
 		}
 
+		/**
+		 * Configures {@code Permissions-Policy} response header.
+		 * @param permissionsPolicy the policy
+		 * @return the {@link PermissionsPolicySpec} to configure
+		 */
+		public PermissionsPolicySpec permissionsPolicy(String permissionsPolicy) {
```

Comment:
This doesn't need to accept any parameters since "permissionsPolicy" is configurable in `PermissionsPolicySpec`.